### ISSUE TITLE
can't parse inversion

### ISSUE THREAD

**Issue by CreadDiscans** [cuthbertLab/music21#384]
some.xml

```
...
<harmony>
  <root>
    <root-step>C</root-step>
  </root>
  <kind>major</kind>
  <inversion>1</inversion>
</harmony>
...

```

```
> s = music21.converter.parse('some.xml')
> s.parts[0].elements[1].elements[4]
<music21.harmony.ChordSymbol C>
> s.parts[0].elements[1].elements[4].inversion()
0
```

I expect the inversion should be 1.

What is wrong?

**Comment by a-papadopoulos** [comment]
This seems to have been fixed in my latest pull request #383 (apparently, it was set and then later unset) -- test added. However, the pitches are not set accordingly. Should I try to fix that @mscuthbert ?

**Comment by mscuthbert** [comment]
Thanks @CreadDiscans -- we'll get this fix in very soon @a-papadopoulos has been working super hard on it!
